pkg/admissioncontroller/webhooks/auth/seed/authorizer.go

```diff
@@ -59,6 +59,7 @@ var (
 	cloudProfileResource           = gardencorev1beta1.Resource("cloudprofiles")
 	configMapResource              = corev1.Resource("configmaps")
 	controllerInstallationResource = gardencorev1beta1.Resource("controllerinstallations")
+	eventResource                  = corev1.Resource("events")
```

Comment:
Yeah, probably, I don't think it's in use yet, but I've added it anyways so that we don't have to care about it in the future in case the client-go internally switches to `events.k8s.io`. See 672331c796d80319d5289df84e020f3f52cb75ea